By John Steele Scott, 15/12/2006
Change the location of the resource file.

--- ./gcc/config/pic30/pic30.h.orig	2009-02-01 09:14:20.000000000 +0100
+++ ./gcc/config/pic30/pic30.h	2009-02-01 09:34:31.000000000 +0100
@@ -215,7 +215,7 @@
 */
 #define ASM_SPEC   "%{!.s:%{!.S:--relax}} %{mcpu=*:-p%*}"
 
-#define CC1_SPEC  "-mresource=%I-../../c30_device.info"
+#define CC1_SPEC  "-mresource=/usr/share/pic30/c30_device.info"
 
 #define LINK_SPEC   "%{mcpu=*:-p%*}"
 
